Formula/g/gabedit.rb

```diff
@@ -25,13 +25,22 @@ class Gabedit < Formula
   end
 
   depends_on "pkg-config" => :build
+
+  depends_on "cairo"
+  depends_on "gdk-pixbuf"
+  depends_on "glib"
   depends_on "gtk+"
   depends_on "gtkglext"
+  depends_on "pango"
 
   on_linux do
     depends_on "autoconf" => :build
     depends_on "automake" => :build
     depends_on "libtool" => :build
+
+    depends_on "at-spi2-core"
+    depends_on "gettext"
+    depends_on "harfbuzz"
```

Comment:
These three are failing linkage on MacOS too.

Reply:
oh, I think it got added in a wrong place. should be macos only